### ISSUE TITLE
I'm fused by the output and target noise.

### ISSUE THREAD

**Issue by TMDTom** [Janspiry/Palette-Image-to-Image-Diffusion-Models#96]
noise = default(noise, lambda: torch.randn_like(y_0))
        y_noisy = self.q_sample(
            y_0=y_0, t=t, noise=noise)

        if mask is not None:
            noise_hat = self.denoise_fn(torch.cat([y_cond, y_noisy*mask+(1.-mask)*y_0], dim=1), t)
            loss = self.loss_fn(mask*noise, mask*noise_hat)
        else:
            noise_hat = self.denoise_fn(torch.cat([y_cond, y_noisy], dim=1), t)
            loss = self.loss_fn(noise, noise_hat)
        return loss
    
The target noise is generated by white guassian noise whose value is [-3 , 3] with the bigger probility. The output activate function is Silu,whose value is [0 1]. I don't understand. Is there anybody can help me?
